src/Domain/Setting/SettingRepositoryInterface.ts

```diff
@@ -1,13 +1,11 @@
 import { DeleteSettingDto } from '../UseCase/DeleteSetting/DeleteSettingDto'
-import { CreateOrReplaceSettingDto } from './CreateOrReplaceSettingDto'
-import { CreateOrReplaceSettingStatus } from './CreateOrReplaceSettingStatus'
 import { Setting } from './Setting'
 
 export interface SettingRepositoryInterface {
   findOneByNameAndUserUuid(name: string, userUuid: string): Promise<Setting | undefined>
   findAllByUserUuid(userUuid: string): Promise<Setting[]>
 
-  createOrReplace(dto: CreateOrReplaceSettingDto): Promise<CreateOrReplaceSettingStatus>
-
   deleteByUserUuid(dto: DeleteSettingDto): Promise<void>
+
```

Comment:
no need for empty lines between methods in interface

Reply:
Here they separate methods that represent different CRUD groups (granted, there is mostly only one per group, but it does no harm, so I'll leave that as is).